deploy/charts/kubernetes-cluster-prep/templates/clusterrole.yaml

```diff
@@ -0,0 +1,28 @@
+{{- if .Values.authnK8s.rbac.create }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ .Values.authnK8s.rbac.clusterRole }}
+  labels:
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    app.kubernetes.io/name: {{ .Release.Name }}
+    app.kubernetes.io/component: conjur-config
```

Comment:
Looking at the Kubernetes doc noted earlier, the `component` is a high level description of where this thing fits in the architecture. So maybe:
```
    app.kubernetes.io/component: authn-k8s-permissions
```
or:
```
    app.kubernetes.io/component: rbac
```